src/cosmos/Swap.fixture.tsx

```diff
@@ -75,31 +75,40 @@ function Fixture() {
 
   const [routerUrl] = useValue('routerUrl', { defaultValue: 'https://api.uniswap.org/v1/' })
 
-  return (
+  const widget = (
+    <SwapWidget
+      convenienceFee={convenienceFee}
+      convenienceFeeRecipient={convenienceFeeRecipient}
+      defaultInputTokenAddress={defaultInputToken}
+      defaultInputAmount={defaultInputAmount}
+      defaultOutputTokenAddress={defaultOutputToken}
+      defaultOutputAmount={defaultOutputAmount}
+      hideConnectionUI={hideConnectionUI}
+      locale={locale}
+      jsonRpcUrlMap={INFURA_NETWORK_URLS}
+      defaultChainId={defaultChainId}
+      provider={connector}
+      theme={theme}
+      tokenList={tokenList}
+      width={width}
+      routerUrl={routerUrl}
+      onConnectWalletClick={useHandleEvent('onConnectWalletClick')}
+      onReviewSwapClick={useHandleEvent('onReviewSwapClick')}
+      onTokenSelectorClick={useHandleEvent('onTokenSelectorClick')}
+      onTxSubmit={useHandleEvent('onTxSubmit')}
+      onTxSuccess={useHandleEvent('onTxSuccess')}
+      onTxFail={useHandleEvent('onTxFail')}
+    />
+  )
+
+  // If iframed, only display the SwapWidget, without any Chrome.
```

Comment:
without any Chrome?

Reply:
chrome (not capitalized) - meaning the extra UI around the content being tested.